lib/include/private/aws_iot_defender_internal.h

```diff
@@ -0,0 +1,81 @@
+#ifndef _AWS_IOT_DEFENDER_INTERNAL_H_
+#define _AWS_IOT_DEFENDER_INTERNAL_H_
+
+/* Build using a config header, if provided. */
+#ifdef AWS_IOT_CONFIG_FILE
+    #include AWS_IOT_CONFIG_FILE
+#endif
+
+/* Defender include. */
+#include "aws_iot_defender.h"
+
+/* constants */
+#define AWS_IOT_DEFENDER_METRICS_FORMAT_CBOR    1
```

Comment:
Maybe make these enums?

Reply:
ok, let me try to figure out how to do it in c